device/jsonld-contexts/device.jsonld

```diff
@@ -1,14 +1,20 @@
 {
     "@context": {
         "Device": "https://smartdatamodels.org/dataModel.Device/Device",
+        "appEUI": "https://smartdatamodels.org/dataModel.Device/appEUI",
         "applicationId": "https://smartdatamodels.org/dataModel.Device/applicationId",
         "batteryVoltage": "https://smartdatamodels.org/dataModel.Device/batteryVoltage",
         "chargerStatus": "https://smartdatamodels.org/dataModel.Device/chargerStatus",
-        "deviceName": "https://smartdatamodels.org/dataModel.Device/deviceName",
+        "description": "https://smartdatamodels.org/dataModel.Device/description",
+        "devEUI": "https://smartdatamodels.org/dataModel.Device/devEUI",
         "deviceType": "https://smartdatamodels.org/dataModel.Device/deviceType",
         "internalHumidity": "https://smartdatamodels.org/dataModel.Device/internalHumidity",
         "internalTemperature": "https://smartdatamodels.org/dataModel.Device/internalTemperature",
+        "isDeleted": "https://smartdatamodels.org/dataModel.Device/isDeleted",
+        "isEnabled": "https://smartdatamodels.org/dataModel.Device/isEnabled",
         "loraSNR": "https://smartdatamodels.org/dataModel.Device/loraSNR",
+        "name": "https://smartdatamodels.org/dataModel.Device/name",

```

Comment:
`name` is usually mapped to "https://schema.org/name" (unless the name of a device is a special kind of name).

Reply:
we can use the regular "https://schema.org/name" in my opinion